### PR TITLE
Only engineering staff have maintenance access

### PR DESCRIPTION
**Pull request by MadmanMartian**
[honk]

In the distant past, maintenance was used for maintaining stations

:cl:
 * tweak: Only engineering staff have maintenance access

### CODE REVIEW

File: code/game/jobs/job/civilian.dm [hunk 1]
```diff
@@ -228,8 +228,8 @@
 	wage_payout = 20
 	selection_color = "#dddddd"
 	idtype = /obj/item/weapon/card/id/supply
-	access = list(access_maint_tunnels, access_mailsorting, access_cargo, access_cargo_bot, access_mint, access_mining, access_mining_station)
```
Comment (Intigracy) [inline code/game/jobs/job/civilian.dm:231]: how else will they crate slave?

**Comment by AI-Is-Rogue** [comment]
Finally an excuse to remove all of the maint tumors and go back to what it was like before the abandoned theatre

**Comment by Accipitermaris** [comment]
Finally, a solution that displeases almost everyone

**Comment by SonixApache** [comment]
make sure you remove maint from mechanics

**Comment by robosperg** [comment]
captain should keep maint

**Comment by ghost** [comment]
captain has no reason to be in maintenance
remove maintenance access from the ID consoles

**Comment by Intigracy** [comment]
Upvoting meme prs is why we can't have nice things and why nobody takes emoji votes seriously

**Comment by tuypo1** [comment]
>captain has no reason to be in maintenance

The captain more than maybe anybody else needs to be able to get anywhere on station quickly, a good captain makes optimal use of maint shortcuts

**Comment by Tokanova** [comment]
if we remove maint access i will start removing maint doors.

**Comment by ShiftyRail** [comment]
Sinking lower and lower in IQ with each passing PR

**Pull request closed without merging**